client/starwhale/core/dataset/cli.py

```diff
@@ -128,6 +128,9 @@ def _diff(
 @click.option(
     "--size", type=int, default=DEFAULT_PAGE_SIZE, help="Page size for dataset list"
 )
+@click.option("-n", "--name", help="Prefix of dataset name")
+@click.option("-o", "--owner", help="[Cloud]Name or id of the dataset owner")
+@click.option("-l", "--latest", is_flag=True, help="Only show the latest version")
```

Comment:
Could we provide a filter option that can describe all filters syntax? for example: https://docs.docker.com/engine/reference/commandline/ps/#filtering

Reply:
done